config/software/ruby.rb

```diff
@@ -26,7 +26,7 @@
 # the default versions should always be the latest release of ruby
 # if you consume this definition it is your responsibility to pin
 # to the desired version of ruby. don't count on this not changing.
-default_version "3.1.3"
+default_version "3.2.2"
```

Comment:
@poorndm should we consider updating default to 3.2.2 only after validating all product builds go through fine? Or do they already override the default in their respective `omnibus_overrides.rb`?

Reply:
@vkarve-chef  chef-server/workstation/inspec/supermarket/chef-manage are overriding the ruby with different versions in their respective omnibus_overrides.rb. Even though we need to validate 3.2.2 for all dependent products.

Reply:
@rfpronk Lets retain the default version to 3.1.x and keep the ruby 3.2.2 in the version list.
yet we have'nt made a plan to pick 3.2.x .

Reply:
Okay I will change it to the latest 3.1
I changed it to 3.2 as I was following the comment right above it,  I will update that as well.